### PR TITLE
E1656. Improve imports

### PR DESCRIPTION
**Pull request by olganam**
This is pull request for: E1656. Improve imports
Project explanation:  http://wiki.expertiza.ncsu.edu/index.php/CSC/ECE_517_Fall_2016/oss_E1656

### CODE REVIEW

File: app/controllers/import_file_controller.rb [hunk 1]
```diff
@@ -31,13 +74,14 @@ def importFile(session, params)
     delimiter = get_delimiter(params)
     file = params['file'].tempfile
 
+    logger.debug "the value of file is passed: #{file}"
```
Comment (yangsong8) [inline app/controllers/import_file_controller.rb]: This should be removed

**Comment by yangsong8** [comment]
I tested your pull request,

- Export participants of assignment 754 with choice "All Paritcipants" and I got a list of students with "All Participants" as "team_id" and "no topic chosen" and topic. This does not look correct. This is wikipedia assignment and most students should have topics.

File: app/controllers/import_file_controller.rb [hunk 2]
```diff
@@ -11,6 +11,49 @@ def start
...
+        when "ReviewResponseMap"
+          @array_expected_values = parse_line("Contributor, Reviewer1, Reviewer2,...,ReviewerN",',',params)
+        when "MetaeviewResponseMap"
+          @array_expected_values = parse_line("Contributor, Reviewer, Metareviewer1,Metareviewer2,...,MetareviewerN",',',params)
+        when "SignUpTopic"
+          @array_expected_values = parse_line("Topic identifier, Topic name, Max choosers, Topic Category",',',params)
+        when "SignUpSheet"
+          @array_expected_values = parse_line("Topic identifier, user1, user2,...",',',params)
+        else
+          @array_expected_values = parse_line("Team Name - optional, Team Member1,Team Member2, ... ,Team MemberN",',',params)
```
Comment (yangsong8) [inline app/controllers/import_file_controller.rb]: What type will fall into this else branch? "AssignmentParticipant", "CourseParticipant" and "CourseTeam" are not covered above, but I don't think they all share the same expected fields.

File: app/controllers/import_file_controller.rb (hunk 1 above)
Comment (olganam) [inline app/controllers/import_file_controller.rb]: Will update. thanks

File: app/controllers/import_file_controller.rb (hunk 2 above)
Comment (olganam) [inline app/controllers/import_file_controller.rb]: Will update. thanks

**Comment by Rishabh2693** [comment]
Will Update the required change in all participants

**Pull request closed without merging**